.circleci/config.yml

```diff
@@ -17,6 +17,8 @@ jobs:
           key: yarn-dependency-cache-{{ checksum "yarn.lock" }}
           paths:
             - node_modules
+      - run: git fetch origin master 
+      - run: yarn commitlint --from $(git merge-base --fork-point FETCH_HEAD) 
```

Comment:
After testing a bit turns out it can be difficult to find which commits are relevant just using `git` locally. I switched to using the PR URL and the compare URL in CircleCI env to find out which commit should be checked.